### ISSUE TITLE
ITS NOT WORKING IN ANDROID 

### ISSUE THREAD

**Issue by johndig0027** [shukerullah/react-native-google-publisher-tag#1]
Hi Team, 
This lib is working perfectly for ios but, i dont know whats the issue... in android its showing me 404 not found page. Could you help me regarding same.

**Comment by shukerullah** [comment]
Hi @johndig0027 
Can share your code if you don't mind and is it working with sample ad units?

**Comment by rahulnakate** [comment]
yes, I have tried with your sample add unit, its working perfectly in ios , but in android i dont know whats the issue. Even you sample also not working in android, Code is nothing different than what you have given.  
` <GooglePublisherTag
          adUnitId="/6499/example/APIDemo/AdSizes"
          adUnitSize="[[320, 50]]"
          baseUrl="https://your-domain.com"
          adUnitTargeting={{
            interest: "basketball",
          }}
        />`

Dont know whats the issue? Could you help me out for this?  even you can refer this, i have tried with direct WebView but still same problem is there 
[https://github.com/react-native-community/react-native-webview/issues/1027](url)

**Comment by shukerullah** [comment]
Hi @rahulnakate,

What baseUrl are you passing? Use your own domain or remove it.

**Comment by rahulnakate** [comment]
Yes i have tried with my own URL as well as removing url , but still same issue is  there. But in ios its working perfectly. Dont know whats the issue in android only. its showing blank only

**Comment by shukerullah** [comment]
Just tested on a new project. It's working fine. Here is screenshot:
<img width="694" alt="Screen Shot 2020-09-07 at 5 08 19 PM" src="https://user-images.githubusercontent.com/963765/92390934-de616000-f12c-11ea-8eed-986cc605d710.png">

Production Screenshot:
![WhatsApp Image 2020-09-07 at 5 10 33 PM](https://user-images.githubusercontent.com/963765/92391086-1a94c080-f12d-11ea-8ddc-14356c7a8338.jpeg)

You can also check in this live [app](https://play.google.com/store/apps/details?id=com.gulfnews.newsapp)

**Comment by rahulnakate** [comment]
OHH...THATS GREAT !!!  But here i have tried it on my project it giving same issue.  may i know what react-native-webview version you are using also possible please tel me whats the react native version. Below are the my version details

     "react-native-webview": "^7.0.5",
    "react": "16.8.6",
    "react-native": "0.60.5", 


Also i have added my Screen shotss 
![android_screenshots](https://user-images.githubusercontent.com/7077797/92393240-343e0400-f13d-11ea-9c6e-b4e4cc3ecc1b.png)
<img width="515" alt="Screenshot 2020-09-07 at 7 02 43 PM" src="https://user-images.githubusercontent.com/7077797/92393249-37d18b00-f13d-11ea-9237-07582d32b2be.png">

But In ios its work perfectly 

![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-09-07 at 19 02 58](https://user-images.githubusercontent.com/7077797/92393450-8da63300-f13d-11ea-8bbc-5e35bb9aff8e.png)

**Comment by shukerullah** [comment]
@rahulnakate can you message me [here](https://www.facebook.com/shukerullah)? I will delete this message

**Comment by glopezs18** [comment]
Did you manage to resolve this issue? I'm new to this and I'm trying to do the same thing but I'm getting the same problem

**Comment by shukerullah** [comment]
> Did you manage to resolve this issue? I'm new to this and I'm trying to do the same thing but I'm getting the same problem

What react-native version and webview are you using? What baseUrl are you passing?

**Comment by glopezs18** [comment]
This is the version I'm using

![Captura de Pantalla 2022-10-12 a la(s) 10 27 44](https://user-images.githubusercontent.com/71115685/195397785-36e92d6a-278d-4b99-998a-8cea2cc5b688.png)
![Captura de Pantalla 2022-10-12 a la(s) 10 28 25](https://user-images.githubusercontent.com/71115685/195397924-96794d13-2297-49dd-ad40-43da5ecc8c6d.png)
